pkg/processtree/processtree.go

```diff
@@ -99,7 +96,7 @@ func New(log *logging.Logger, p *proc.Proc, containersClient containerClient) (*
 		log:              log,
 		proc:             p,
 		containersClient: containersClient,
-		eventSink:        make(chan ProcessTreeEvent, 1000),
+		eventSink:        make(chan ProcessTreeEvent, 100),
```

Comment:
As currently the process tree init runs in sync, we would loose process tree events if there are more than `100` processes, which is very likely.

Reply:
Since only Init is left it should fire only once because it collects all processes and sends in a single batch. 
Even 100 chan size is too big, but maybe let's add a todo to to have a simple blocking Init and send data without channels?

Reply:
Init is already sync. What is happening right now is that the whole initial process tree is buffered into the channel. As said, it is very likely that there are more than 100 processes running on the host we would create a snapshot for, causing us to miss processes. Additionally, this channel isn't doing anything after launch just sitting around.